main.js

```diff
@@ -281,6 +282,113 @@ function checkOldInstallation(){
   })
 }
 
+// delete directory
+function rmdir(dir,cb) {
+  fs.readdir(dir,function (err, files) {
+    if (err) {
+      console.error(err)
+    } else {
+      next(0);
+      function next(index) {
+        if(index == files.length) {
+          return fs.rmdir(dir,cb);
+        }
+        let newPath = path.join(dir,files[index]);
+        fs.stat(newPath,function (err, stats) {
+          if(err){
+            console.error(err);
+          }
+          if(stats && stats.isDirectory()){
+            rmdir(newPath,()=>next(index+1));
+          } else {
+            fs.unlink(newPath,function (err) {
+              if (err) {
+                console.error(err);
+              }
+              next(index + 1);
+            });
+          }
+        })
+      }
+    }
+  })
+}
+
+// remove directory
+async function removeDir(path) {
+  return new Promise((resolve, reject) => {
+    rmdir(path, () => {
+      if (fs.existsSync(path)) {
+        // delete soft links
+        fs.readdir(path, (err, files) => {
+          if (err) {
+            console.error(err)
+            reject(err)
+          } else {
+            rmdir(path ,() => {
```

Comment:
I think the here it should not delete `path` right? There should be a for loop to delete files right:
```
for(let f of files){
  if(f_is_directory){
    rmdir(f)
  }
  else{
    fs.unlinkSync(f)
  }
}
```